modules/configuring-log-storage-cr.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploying.adoc
+// * logging/cluster-logging-loki.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-log-storage-cr_{context}"]
+= Configuring log storage
+
+You can configure which log storage type your {logging} uses by modifying the `ClusterLogging` custom resource (CR).
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo} and an internal log store that is either the LokiStack or Elasticsearch.
```

Comment:
Should `OpenShift Elasticsearch Operator` be the `OpenShift Container Platform Elasticsearch Operator`

Reply:
OpenShift Elasticsearch Operator is the literal name of the Operator: 
![image](https://github.com/openshift/openshift-docs/assets/40172997/8da9a24e-3caa-47b9-9aa3-3deddad9a71a)